css/amp.css

```diff
@@ -153,6 +153,14 @@ html.i-amphtml-ios-embed.i-amphtml-ios-overscroll > #i-amphtml-wrapper {
   position: relative;
 }
 
+/**
+ * Children of responsive elements should not be able to influence the height
+ * which they do when they are statically positioned.
+ */
+.i-amphtml-layout-responsive > :not(i-amphtml-sizer) {
```

Comment:
The problem (and maybe there is a better way to address it is) that there is a phase between stub-build and final-build that isn't matched by `.i-amphtml-notbuilt:not(.i-amphtml-layout-container) > *`